packages/rmf-auth/package.json

```diff
@@ -2,25 +2,27 @@
   "name": "rmf-auth",
   "version": "0.0.1",
   "description": "Auth layer to be used on web UI services",
-  "main": "lib/index.js",
+  "main": "index.js",
   "scripts": {
-    "start": "start-storybook -p 6006 -s stories",
-    "prepare": "npm run build",
+    "start": "start-storybook -p 6006 -s test-data",
+    "prepare": "tsc --build && cp package.json dist/",
```

Comment:
This is useful.
The pipeline is failing because the build command is missing, should we replace it with prepare?